packages/web/src/components/gcds-file-uploader/gcds-file-uploader.stories.tsx

```diff
@@ -62,48 +62,48 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-file-uploader
   uploader-id="${args.uploaderId}"
   label="${args.label}"
```

Comment:
Label is not marked as required in the story and it also doesn't have a data type defined. Do you mind adding that in?